causal_testing/testing/estimators.py

```diff
@@ -65,6 +70,15 @@ class LinearRegressionEstimator(Estimator):
     """ A Linear Regression Estimator is a parametric estimator which restricts the variables in the data to a linear
     combination of parameters and functions of the variables (note these functions need not be linear).
     """
+    def __init__(self, treatment: tuple, treatment_values: float, control_values: float, adjustment_set: set,
+                 outcome: tuple, df: pd.DataFrame = None, effect_modifiers: {Variable: any} = None, product_terms: [(Variable, Variable)] = None):
```

Comment:
Quick question about [(Variable, Variable)]. Is it a list of tuples, if so I'm not sure what the two "Variable"s represent